README.md

```diff
@@ -273,6 +273,33 @@ withInfraPipeline(product) {
 }
 ```
 
+#### Optional parameters for the opinionated infratructure pipeline
+
+You have the ability to pass extra parameters to the `withInfraPipeline`.
+
+These parameters include:
+| parameter name | description | example |
+| --- | --- | --- |
+| component | Extra detail to the name of the repository | "extra-detail" |
```

Comment:
not the best name really for the description, wondering if should just link to the glossary?
https://hmcts.github.io/glossary/#component